app/services/bookings/gitis/entity.rb

```diff
@@ -65,7 +65,9 @@ def entity_id
     end
 
     def attributes_for_update
-      attributes.slice(*(changed - update_blacklist))
+      attributes.slice(*(changed - update_blacklist)).reject do |k, v|
+        k =~ /@odata\.bind\z/ && v.nil?
```

Comment:
`#end_with?` might have been clearer here but not really too important

Reply:
Yeh - good call - will update the PR and merge, thanks